spec/spec_helper.rb

```diff
@@ -16,4 +18,5 @@
   config.warnings = true
 
   Dir[File.expand_path('support/**/*.rb', __dir__)].each { |f| require f }
+  FileUtils.mkdir_p('tmp')
```

Comment:
This could add either the process ID or a random string so test runs won't interfere with one another.

The temporary should be removed after the full test runs.